modules/TaskFormPresenter.js

```diff
@@ -0,0 +1,41 @@
+class TaskFormPresenter {
+  #input;
+
+  #submit;
+
+  #view;
+
+  constructor() {
+    this.#view = document.createElement('form');
+    this.#input = document.createElement('input');
+    this.#input.setAttribute('placeholder', 'Add to your list...');
+    this.#input.setAttribute('id', 'new-item');
+    this.#input.setAttribute('type', 'text');
+    this.#view.appendChild(this.#input);
+    this.#submit = document.createElement('input');
+    this.#submit.setAttribute('class', 'icon-return');
+    this.#submit.setAttribute('id', 'submit-new-item');
+    this.#submit.setAttribute('type', 'submit');
+    this.#submit.setAttribute('value', '');
+    this.#submit.setAttribute('title', 'click this or press enter to submit');
+    this.#view.appendChild(this.#submit);
+    this.#view.addEventListener('submit', this.#submitTask);
```

Comment:
Some of these variables are used only in one method, so please try to create these variables as local variables in the method that uses them.

Reply:
Thank you!